main.tf

```diff
@@ -0,0 +1,45 @@
+data "template_file" "body" {
+  template = "${var.body}"
+
+  vars = "${var.vars}"
+}
+
+data "template_file" "subject" {
+  template = "${var.subject}"
+
+  vars = "${var.vars}"
+}
+
+locals {
+  body    = "${data.template_file.body.rendered}"
+  subject = "${data.template_file.subject.rendered}"
+  command = "${var.mail_command} ${join(" ", var.to)}"
+}
+
+resource "null_resource" "default" {
+  count = "${var.enabled == "true" ? 1 : 0}"
+
+  triggers = {
+    subject = "${local.subject}"
+    body    = "${local.body}"
+    command = "${local.command}"
+  }
+
+  provisioner "local-exec" {
+    command = "${local.command}"
+
+    environment {
```

Comment:
is this specific to this tool https://github.com/wrouesnel/emailcli?

Reply:
Yes